subsys/spm/secure_services_ns.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <zephyr.h>
+#include <errno.h>
+#include <../arch/arm/include/aarch32/cortex_m/tz_ns.h>
+#include <fw_info.h>
+
+void spm_request_system_reboot_nsc(void);
```

Comment:
Why are these needed? Would it be possible to get rid of them by adding a new layer of macros?

Reply:
Done